inception-example-imls-data-majority/src/test/java/de/tudarmstadt/ukp/inception/recommendation/imls/datamajority/DataMajorityNerRecommenderTest.java

```diff
@@ -98,7 +99,12 @@ public void thatPredictionWorks() throws Exception
 
         sut.predict(context, cas);
 
-        Collection<NamedEntity> predictions = getPredictions(cas, NamedEntity.class);
+        Collection<NamedEntity> predictions = null;
+        try {
+            predictions = getPredictions(cas, NamedEntity.class);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
```

Comment:
Catching exception and wrapping it in a runtime exception doesn't really contribute anything here. It just means that Exception doesn't appear on the `throws` statement. It's not a proper handling. I prefer just having `throws Exception` and that's it.

Reply:
This applies to other changes in this PR likewise.